### PR TITLE
PLA-4107: Changed cis sorting

### PR DESCRIPTION
**Pull request by emilkip**

### CODE REVIEW

**Comment by dgumennikov** [comment]
Do not merge please. We need to finish this task at first CON-578. And then check it altogether.

**Comment by faustest** [comment]
test this please

**Comment by dgumennikov** [comment]
retest this please

File: js/utils.js [hunk 1]
```diff
@@ -45,34 +45,23 @@ window.utils = {
...
+                idB[i] = parseInt(idB[i]);
 
-                if (idA.first > idB.first) return 1;
-                else if ((idA.first >= idB.first) && idA.second > idB.second) return 1;
-                else return -1;
+                if (idA[i] === idB[i]) continue;
+                if (idA[i] > idB[i]) return 1;
+                return -1;
             }
+            return 1;
```
Comment (enigma91) [inline js/utils.js]: you can not return 1 you should return (idA.length > idB.length) ? 1 : -1

**Pull request merged**